src/formatter/formatReactElementNode.js

```diff
@@ -126,13 +127,15 @@ export default (
 
   const visibleAttributeNames = [];
 
+  const propFilter = createPropFilter(props, filterProps);
```

Comment:
The `propFilter` variable could be renamed `filterOutBlacklistedProps` to be more explicit on what it does

Reply:
I can change it, although I don't think it's any more explicit really. With the function form there isn't a list of any sort,  and even then it's more of a "whitelist" since you return `true` to keep the value.